### PR TITLE
Don't state crashlooping container when pod only has one container

### LINKED ISSUE #9437: crashlooping container in oc status
If there is only one container in the pod then we should show something like:
```
  * pod/database-3-hook-pre is crash-looping
```
instead of:
```
  * container "lifecycle" in pod/database-3-hook-pre is crash-looping
```
@deads2k thoughts?

### PR DESCRIPTION
**Pull request by enj**
fixes #9437 

Tests are a bit ugly...

### CODE REVIEW

File: pkg/api/kubegraph/analysis/pod_test.go [hunk 1]
```diff
@@ -38,6 +39,20 @@ func TestRestartingPodWarning(t *testing.T) {
 		t.Fatalf("expected %v, got %v", e, a)
 	}
 
+	sort.Sort(osgraph.ByNodeID(markers))
+	if strings.Index(markers[0].Message, "container ") != 0 {
```
Comment (deads2k) [inline pkg/api/kubegraph/analysis/pod_test.go]: `strings.Contains`

Comment (enj) [inline pkg/api/kubegraph/analysis/pod_test.go]: I was trying to do something like `strings.StartsWith` in case the word `container` occurs elsewhere.

Comment (liggitt) [inline pkg/api/kubegraph/analysis/pod_test.go]: then do strings.HasPrefix :)

**Comment by kargakis** [comment]
@enj can you also do the same for container restarts? Same file, different warning.

**Comment by enj** [comment]
@kargakis I thought I already did that?

**Comment by kargakis** [comment]
> @kargakis I thought I already did that?

Oh, sorry didn't review - just inferred what you did from the PR title

**Comment by deads2k** [comment]
lgtm [merge]

**Comment by enj** [comment]
**Alakazam** [test]

**Comment by kargakis** [comment]
[merge]

**Pull request merged**